src/components/FoodDetails.js

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+
+
+
+function FoodDetails(props) {
+
+/*
+  props = {
```

Comment:
to work around a state like this you will need to use the hooks syntax or change it into a class component.